symplyphysics/laws/thermodynamics/equations_of_state/van_der_waals/dimensionless_equation.py

```diff
@@ -30,7 +33,33 @@
     Rational(8, 3) * reduced_temperature,
 )
 
-# TODO: derive from van der Waals equation of state
+# Derive from van der Waals equation of state
+
+_critical_pressure_expr = critical_pressure.law.rhs.subs({
+    critical_pressure.bonding_forces_parameter: equation.bonding_forces_parameter,
+    critical_pressure.molecules_volume_parameter: equation.molecules_volume_parameter,
+})
+
+_critical_molar_volume_expr = critical_molar_volume.law.rhs.subs({
+    critical_molar_volume.molecules_volume_parameter: equation.molecules_volume_parameter,
+})
+
+_critical_temperature_expr = critical_temperature.law.rhs.subs({
+    critical_temperature.bonding_forces_parameter: equation.bonding_forces_parameter,
+    critical_temperature.molecules_volume_parameter: equation.molecules_volume_parameter,
+})
+
+_reduced_eqn = equation.law.subs({
```

Comment:
I tried to pull it off with equations for reduced quantities and then plugging it into `solve` so that `sympy` performed the necessary substitutions, but nothing seemed to trigger them